grakn-debian

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
```

Comment:
do we really need `grakn-debian`? I wonder if we can just use the existing`grakn`.

Reply:
@lolski fixed in https://github.com/graknlabs/grakn/pull/4775/commits/eb01ae4f3977747ac1c36cf1f1071530f342e7ea